README.md

```diff
@@ -94,13 +94,6 @@ yarn
 yarn build
 ```
 
-Alternatively, to continuously watch the frontend assets and rebuild as necessary:
```

Comment:
Is there no way to maintain this behavior?

Reply:
Added back in https://github.com/cfpb/website-indexer/pull/119/commits/42dad7965571129386e7516a5521163aba6bfc43